### ISSUE TITLE
Popover: change the Rich Interactive Tooltip sample to use a "toggle-tip" instead

### ISSUE THREAD

**Issue by rolfsmeds** [vaadin/docs#5766]
The [Rich Interactive Tooltip sampe of Popover](https://vaadin.com/docs/latest/components/popover#rich-interactive-tooltip) is not announced by screen readers because `<input type="text">` does not support `aria-expanded` or `aria-haspopup`.

Modify the sample to be a "toggle-tip" instead, i.e. add a suffix (i) button to the field that opens the popover, and modify the description accofdingly.
